src/src/App.spec.tsx

```diff
@@ -0,0 +1,8 @@
+import { shallow } from "enzyme";
+import React from "react";
+import App from "./App";
+
+it("renders without crashing", () => {
+  const component = shallow(<App />);
+  expect(component).toMatchSnapshot();
```

Comment:
This test will actually fail every time you change **any** visual thing in your app. Personally I use `toMatchSnapshot` only on "dumb" components that are rendering visual stuff and that all. (Most of the times they are very thin)